README.md

```diff
@@ -55,23 +55,23 @@ python setup.py install
 ```bash
 # Step 1: Download metagpt official image and prepare config.yaml
 docker pull metagpt/metagpt:v0.2
-mkdir -p /opt/metagpt/{config,workspace} && chmod 777 -R /opt/metagpt
-docker run --rm metagpt/metagpt:v0.2 cat /app/metagpt/config/config.yaml > /opt/metagpt/config/config.yaml
+mkdir -p ./opt/metagpt/{config,workspace} && chmod 777 -R ./opt/metagpt
```

Comment:
`chmod 777 -R /opt/metagpt` can be deleted, because the new image can use the root user.